web-react/src/pages/directory/quick-facts/QuickFacts.css

```diff
@@ -0,0 +1,158 @@
+:root {
+  --background-base: #121212;
+  --apple-cyan: #0bc9d2;
+  --text-blue: #68e0d7;
+  --lighter-card: #242323;
+  --text-red: #da0218;
+  ---ffffff: #ffffff;
+  --card-grey: #1e1c1c;
+  --text-green: #4de531;
+  --text-grey: #8e8c8c;
+  --unnamed-color-4ae545: #4ae545;
```

Comment:
And I don't know about these unnamed colours. Please change them

Reply:
i have removed this as well